ArchiSteamFarm/NLog/Logging.cs

```diff
@@ -206,6 +206,11 @@ internal static void EnableTraceLogging() {
 	}
 
 	internal static void InitCoreLoggers(bool uniqueInstance) {
+		if (LogManager.Configuration == null) {
```

Comment:
Should this check not be moved below the logic for explicit loading `XmlLoggingConfiguration` ?